homeassistant/components/flexpool/strings.json

```
@@ -0,0 +1,18 @@
+{
+    "config": {
+        "error": {
+            "invalid_address": "Address is invalid",
+            "unknown": "[%key:common::config_flow::error::unknown%]"
+        },
+        "step": {
+            "user": {
+                "data": {
+                    "address": "Address",
+                    "pool": "Get Pool Stats",
+                    "workers": "Get Workers"
+                }
+            }
+        }
+    },
+    "title": "Flexpool"
```

The title doesn't need to be translated, right?

```suggestion
    }
```